app/decorators/submission_trait_scores_upload_decorator.rb

```diff
@@ -12,14 +12,47 @@ def parser_summary
           group_by{ |id, scores| scores.size }.
           sort{ |score1, score2| score2.first <=> score1.first }
 
-        summary << " - parsed #{trait_scores.size} plant(s) with unique identification"
+        summary << " - parsed #{trait_scores.size} plant scoring unit(s) with unique identification"
         histogram.each do |scoring_number, plants|
-          summary << "  - #{plants.size} plant(s) have #{scoring_number} trait score(s) recorded"
+          summary << "  - #{plants.size} unit(s) have #{scoring_number} trait score(s) recorded"
         end
       end
 
+      accessions = object.submission.content.step04.accessions
```

Comment:
What I (really ;)) meant was "Let's not do that right now, as we have a lot of other stuff to write/test, but in case we are asked for considerable submission process extensions in the future, we should address it somehow.".

If Wiktor's plans work (we hope they do) and BIP starts to be widely used by plant experimentalists, we will most probably go back to that issue some day.